eng/CredScanSuppression.json

```diff
@@ -25,6 +25,10 @@
       "hash": "5ETIzLYe9aa9Xx5512Uy6gJiCZ/CL2QzS3nDfN51nz8=",
       "_justification": "Secret used by test code"
     },
+    {
+      "placeholder": "SharedKey fakestorageaccount:pass123",
```

Comment:
Any reason to not add it to the list of placeholders? It looks like there are others in this file that we should probably also include in the one list as well.

Also

Reply:
Yes I can combine them. I separate this to have a different `_justification` but that is not so important.